.gitignore

```diff
@@ -3,7 +3,6 @@ node_modules
 **/*.map
 docs
 build
-!packages/proxyIdentity/build
```

Comment:
Why including `packages/proxyIdentity/build` now?

Reply:
I propose following approach chosen for iam-contracts, namely building artifacts when installing package rather then storing them in repository